components/dashboard/sections/overview/Metric.tsx

```diff
@@ -42,9 +42,10 @@ export interface MetricProps extends React.ComponentPropsWithoutRef<'div'> {
   };
   helpLabel?: React.ReactNode;
   showCurrencyCode?: boolean;
+  showTimeSeries?: boolean;
   expanded?: boolean;
-  withTimeseries?: boolean;
   currency?: Currency;
+  isSnapshot?: boolean;
 }
 export function Metric({
```

Comment:
From the creator of the tilting Virtual Card 🤐, I think we should replace the hover:scale animation with a simpler border color update.

Reply:
Makes sense, the scaling is a bit jarring. I removed the scaling and added a ring/border on `focus-visible`, and keeping the shadow for hover.